src/SFA.DAS.Roatp.CourseManagement.Web/Models/ProviderLocations/ProviderLocationViewModel.cs

```diff
@@ -9,9 +9,15 @@ public class ProviderLocationViewModel
         public int? RegionId { get; set; }
         public string LocationName { get; set; }
         public string VenueNameUrl { get; set; }
+        public string AddressLine1 { get; set; }
```

Comment:
This view model was specific to the list. I think we  should leave this as is and have a details view model specifically.